src/bsoncxx/builder/basic/impl.hpp

```diff
@@ -57,6 +60,7 @@ generic_append(core* core, T&& t) {
 }  // namespace
 
 template <typename T>
+BSONCXX_INLINE
```

Comment:
i kind of like having it on its own line for templates so it doesn't distract from the template types.